cleanslate.css

```diff
@@ -354,6 +346,16 @@
 .cleanslate input[type=radio] {
     margin: 2px 2px 3px 2px !important;
 }
+.cleanslate input[type=submit]:active, .cleanslate button:active {
```

Comment:
.cleanslate input[type=button]:active and .cleanslate button:active were removed here.